packages/client/App.tsx

```diff
@@ -53,12 +65,13 @@ const client = new ApolloClient({
 
 export type RootStackParamList = {
   MapView : Parameters<typeof MapView>[0]
-  AccountView : Parameters<typeof AccountView>[0]
+  AccountView : undefined
   PasswordView : Parameters<typeof PasswordView>
   PPView : Parameters<typeof PPView>
   ToSView : Parameters<typeof ToSView>
   HelpView : Parameters<typeof HelpView>
   BusinessView : BusinessViewProps
+  FutureProofRatingView : BusinessViewProps
```

Comment:
This type is incorrect, FutureProofRatingView and BusinessView have different props.

Reply:
The same Business type is being passed into both components though?